### PR TITLE
doc: Initial documentation review

### PR DESCRIPTION
**Pull request by nicklem**
Initial documentation review and placeholder cleanup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +15,7 @@ Before installing the extension, make sure you meet the following requirements:
 1. You have a [Codacy account](https://www.codacy.com/signup-codacy).
 2. The repository you’re working on has been [added to Codacy Cloud](https://docs.codacy.com/organizations/managing-repositories/#adding-a-repository).
 3. You have at least [Repository Read permissions](https://docs.codacy.com/organizations/roles-and-permissions-for-organizations/) for the repository you’re working on.
-4. IntelliJ IDEA, version 2023.3.3 or later.
+4. You're using IntelliJ IDEA version 2023.3.3 or later.
```
Comment (nicklem) [inline README.md:18]: If this matches the [version in gradle.properties](https://github.com/codacy/codacy-intellij-extension/blob/f0ea50985e5c99d3d8a50460ade89ca405e7f3a4/gradle.properties#L15), we can remove this line, as the prerequisite should be checked during installation automatically.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,3 @@
-[//]: # (TODO: technical writers)
 # Codacy IntelliJ IDEA Extension
 
 The Codacy extension for IntelliJ IDEA helps you review and manage the issues found by Codacy directly within the IntelliJ IDE. It notifies you whether a pull request is up to standards by highlighting problematic code patterns and displaying code quality metrics.
```
Comment (nicklem) [inline README.md:3]: Is this for IntelliJ IDEA only? I tried installing it on other platforms (specifically [Writerside](https://www.jetbrains.com/writerside/)) and it seems to work OK. Any limitations on this?

File: README.md [hunk 3]
```diff
@@ -13,37 +12,36 @@ The Codacy extension for IntelliJ IDEA helps you review and manage the issues fo
...
 Before installing the extension, make sure you meet the following requirements:
 
-1. You have a [Codacy account](https://www.codacy.com/signup-codacy).
-2. The repository you’re working on has been [added to Codacy Cloud](https://docs.codacy.com/organizations/managing-repositories/#adding-a-repository).
-3. You have at least [Repository Read permissions](https://docs.codacy.com/organizations/roles-and-permissions-for-organizations/) for the repository you’re working on.
-4. IntelliJ IDEA, version 2023.3.3 or later.
+1.  You have a [Codacy account](https://www.codacy.com/signup-codacy).
+2.  The repository you’re working on has been [added to Codacy Cloud](https://docs.codacy.com/organizations/managing-repositories/#adding-a-repository).
+3.  You have at least [Repository Read permissions](https://docs.codacy.com/organizations/roles-and-permissions-for-organizations/) for the repository you’re working on.
+4.  You're using IntelliJ IDEA version 2023.3.3 or later.
```
Comment (nicklem) [inline README.md:18]: I'm not a fan of hardcoding version numbers unless strictly necessary.

If this is the same version mentioned in [gradle.properties](https://github.com/codacy/codacy-intellij-extension/blob/master/gradle.properties#L15) and the IDE check this when installing, I suggest removing this line.

Or are there other use cases I'm not aware of that justify this?

File: README.md (hunk 2 above)
Comment (alerizzo) [reply README.md:3]: it "should" work for all intellij platform IDEs; eventually we can take a look into it, but I think we also tried it on webstorm and it worked (but installing the zip package)

File: README.md (hunk 3 above)
Comment (alerizzo) [reply README.md:18]: We don't have it for VSCode, no reason to have it here. Feel free to remove it.

File: src/main/resources/inspectionDescriptions/Codacy.html [hunk 4]
```diff
@@ -1,28 +1,29 @@
...
-// automatically highlighted according to inspection registration 'language' attribute
-</code></pre>
-<!-- tooltip end -->
-<p>Text after this comment will only be shown in the settings of the inspection.</p>
 
-<p>To open related settings directly from the description, add a link with `settings://$` optionally followed by `?$` to
-    pre-select a UI element.</p>
+<ol>
+    <li>
+        You have a <a href="https://www.codacy.com/signup-codacy">Codacy account</a>.
```
Comment (alerizzo) [inline src/main/resources/inspectionDescriptions/Codacy.html:18]:
```suggestion
        You have a <a href="https://www.codacy.com">Codacy account</a>.
```

I'd rather go to the public root, not a specific page (not sure who chose `signup-codacy` but doesn't look like something that will endure the test of time)

**Review by alerizzo — APPROVED** [review]

**Pull request merged**